validator/lib/validateTileset.js

```diff
@@ -32,6 +37,71 @@ function validateNode(root, parent, resolve) {
         var tile = node.node;
         parent = node.parent;
 
+        if (defined(tile.content)) {
+            if (defined(tile.content.boundingVolume)) {
+                var region = tile.content.boundingVolume.region;
+                var parentRegion = tile.boundingVolume.region;
+                for (var i = 0; i < region.length; i++) {
+                    if (region[i] > parentRegion[i]) {
+                        return resolve({
+                            result: false,
+                            message: 'Child occupies region greater than parent'
+                        });
+                    }
+                }
+            }
+        }
+
+        if (defined(tile.content) && defined(tile.content.url)) {
+            var tileBuffer = readTile(tile.content.url);
```

Comment:
`readTile` returns a promise.

To handle multiple promises you can create an array at the top of `validateNode` and push the promises to this. Then at the end of the function you can use `Promise.all(tilePromises)` which inside it can return:

```
    return resolve({
        result : true,
        message : 'Tileset is valid'
    });
```